src/profiles/abi_music/parsing.py

```diff
@@ -83,10 +107,13 @@ def parse_project_info(json_data: dict[str, Any]) -> RawProject:
     return raw_project
 
 
-def parse_experiment_info(json_data: dict[str, Any]) -> RawExperiment:
+def parse_experiment_info(directory: DirectoryNode) -> RawExperiment:
     """
     Extract experiment metadata from JSON content
     """
+
+    json_data = read_json(directory.file("experiment.json"))
+
     raw_experiment = RawExperiment(
         title=json_data["experiment_name"],
         description=json_data["experiment_description"],
```

Comment:
I'd check how the identifiers are generated. I think there may be a difference in the experiment IDs in the Raw Experiments compared to the Dataset - IIRC the experiment identifiers are Project-Experiment rather than just the Experiment